### ISSUE TITLE
Add coqc option to make `par:` behave like `all:` (eg -async-proofs-tac-j 0)

### ISSUE THREAD

**Issue by samuelgruetter** [coq/coq#9550]
Following up on https://github.com/coq/coq/issues/9533: In an environment where resources are scarce, it should be possible to run proof scripts using the goal selector `par:` as if this was `all:`, i.e. don't spawn any worker processes, but do the work on the main process.

This behavior could be triggered by `-async-proofs-tac-j 0` (or `-async-proofs-tac-j 1` might make sense as well).

**Comment by gares** [comment]
I agree it makes sense. As of today the implementation of `par:` is quite ugly since it is done at the STM layer while it should be at the tactic layer. I hence think this will have to wait for the ongoing STM cleanup.
